FlowCrypt/Functionality/Error Handling/Task+Retry.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Task+Retry.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 3/30/22
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+extension Task where Failure == Error {
+    @discardableResult
+    static func retrying(
+        priority: TaskPriority? = nil,
+        maxRetryCount: Int = 3,
```

Comment:
Let's do a default of 2. So three API calls total maximum. Current setting with 3 may cause up to 4 API calls which is a bit too much.

Reply:
OK